source/2019-12-20-the-ember-times-issue-129.md

```diff
@@ -0,0 +1,119 @@
+ember-apollo-client v2 released 🎉
+and join us for the last 11 days of DecEmber 1️⃣1️⃣!
+
+READMORE
+
+---
+
+## [Octane Is Here 🎉](https://blog.emberjs.com/2019/12/20/octane-is-here.html)
+
+[Ember 3.15](https://blog.emberjs.com/2019/12/20/ember-3-15-released.html) was released this week. In addition to the usual new features, improvements and bug fixes, this release also marks the official release of Ember's first edition - **Ember Octane**!
+
+But what is Ember Octane? Ember Octane is the **new, recommended way** for developers **to build Ember applications** and **addons**.
+
+<!--alex ignore easy-->
+Octane features a new reactivity model, a renewed practice of building components and a stronger focus on HTML driven web development than ever before. It aims to make the development of performant Ember apps not only more easy and productive, but also **more fun** right from the start. And on top of that, Ember Octane is **fully opt-in** and **interoperable** for existing Ember apps - which means you can try out the new way of building modern Ember Octane apps gradually, without having to rewrite your entire app today.
+
+Want to know all about Ember Octane? Learn more about what's new in Octane and how to get started [in the official Ember Octane release announcement](https://blog.emberjs.com/2019/12/20/octane-is-here.html) by [Yehuda Katz (@wycats)](https://github.com/wycats)! Need help? Check out the **#topic-octane-migration** channel on the [Ember Discord](https://discordapp.com/invite/emberjs)!
+
+---
+
+## [New ember-autofocus-modifier 💡](https://medium.com/qonto-engineering/autofocus-modifier-improving-our-component-apis-fd589b98f4e)
+
+While building an autofocus feature for their components, Qonto engineers were also able to improve their APIs. [Cyrille David (@dcyriller)](https://github.com/dcyriller) and team wanted to add autofocus to several components across two apps. For example, both their one-time-password input and their email input component needed autofocus. Since mixins are no longer recommended in the Ember ecosystem, Cyrille went with modifiers. [And they blogged about their experience here!](https://medium.com/qonto-engineering/autofocus-modifier-improving-our-component-apis-fd589b98f4e)
+
+Modifiers are a new feature in [the recently released Ember Octane edition](https://blog.emberjs.com/2019/12/20/ember-3-15-released.html))https://blog.emberjs.com/2019/12/20/ember-3-15-released.html). They allow for **sharing code between components**, similar to directives in Vue or Angular. We revisited [Chris Garrett (@pzuraq)](https://github.com/pzuraq)'s blog [Coming Soon in Ember Octane - Part 4: Modifiers](https://blog.emberjs.com/2019/03/06/coming-soon-in-ember-octane-part-4.html) and the [Octane Guides preview docs: Out-of-Component-Modifications](https://octane-guides-preview.emberjs.com/release/components/template-lifecycle-dom-and-modifiers/#toc_out-of-component-modifications) to learn more!
```

Comment:
There seems to be a syntax error in the first link (also I would suggest linking to the octane post rather than the 3.15 post). We can also update the preview link to the actual guides url now.

Reply:
Good catch, I updated both accordingly :+1: